src/utils/handlers/farmerHandlers.js

```diff
@@ -108,3 +113,10 @@ export const deleteFarmerHandler = farmerId => {
       throw new Error(error);
     });
 };
+
+export const uploadImageHandler = (imageFile) => {
```

Comment:
add .then and . catch here. Or is there a good reason you left it out?

Reply:
Frankly, I hadn't taken a look at that part of the code since it came in at the very end through merging. Will take a look at it.

Reply:
If I see that correecty ther is a merge conflict now which adds .then and .catch to this handler. Please re-merge and then this PR is good to go IMO. Good job!

Reply:
nvm the last comment, it is just not shown here